dev_setup.sh

```diff
@@ -174,6 +174,30 @@ This script is designed to make working with Mycroft easy.  During this
 first run of dev_setup we will ask you a few questions to help setup
 your environment.'
     sleep 0.5
+    if ! grep -q avx /proc/cpuinfo && [[ ! $(uname -m) == 'arm'* ]]; then
```

Comment:
Forgive me but I am not sure of the objective. I understand the check for avx via cpuinfo but must admit the arm check is confusing me. What is going on here?

Reply:
As eltocino replied below, however it is the inevitable question - so I've added a comment to explain for future devs.

The AVX instruction set is an x86 construct. ARM has a range of equivalents, but I'm not sure which are supported or not by TF. So this is intended to catch the bulk of the support issues we get which is from people trying to install Mycroft on older or low-end x86 hardware.